### DISCUSSION TITLE
How to change inference threshold of fastflow(0.3.3)

### DISCUSSION THREAD

**Discussion by jeaho322** [open-edge-platform/anomalib#1433] [Q&A] [answered]
### What is the motivation for this task?
Hello!, Now, I use fastflow model (anomalib 0.3.3), I use this model for working, so I can't update anomalib library 
so my config.yaml file don't have methed paramters (adaptive/manual) , but only adaptive parameter (true or false)
So I'm trying to change inference threshold to save pred_mask images when predict score is lower than threshold (0,5)
but I couldn't find out. 

### Describe the solution you'd like

so can I know how to change threshold in inference?
In advance, thank you for your help, this model is so helpful to me

**Comment by blaz-r** [comment, marked as answer]
Hello.
Looking at version 0.3.3 I think you could change the `predict_step` function to manually set the threshold here:
https://github.com/openvinotoolkit/anomalib/blob/c0c4cbd4551a97b68705757b83f9a913f5ff4d14/anomalib/models/components/base/anomaly_module.py#L79-L98
Hope this helps.
